pageserver/src/tenant/timeline/compaction.rs

```diff
@@ -709,7 +709,7 @@ impl Timeline {
                     .extend(sparse_partitioning.into_dense().parts);
 
                 // 3. Create new image layers for partitions that have been modified "enough".
-                let image_layers = self
+                let (image_layers, is_complete) = self
```

Comment:
nit: maybe `status` or `outcome` or something now that this is an enum rather than a `bool`.